spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/AwsSecretsManagerEnvironmentRepository.java

```diff
@@ -86,6 +87,10 @@ public Environment findOne(String application, String profileList, String label)
 			label = defaultLabel;
 		}
 
+		if (ignoreLabel) {
```

Comment:
This can be combined with the `if` statement above

Reply:
The above `if` statement is checking `label` provided by request and setting it to `defaultLabel` if empty. This statement set `label` to null regardless the value of `label` or `defaultLabel`.

Reply:
My thought was why set `label` if we are going to ignore the label anyways?

Reply:
Understand. I've updated the code.